src/lighteval/models/vllm/vllm_model.py

```diff
@@ -184,7 +185,7 @@ def _create_auto_model(self, config: VLLMModelConfig, env_config: EnvConfig) ->
             "seed": 1234,
         }
         if int(config.data_parallel_size) > 1:
-            self.model_args["worker_use_ray"] = True
+            self.model_args["distributed_executor_backend"] = "ray"
```

Comment:
Will this work with both versions of vllm?

Reply:
It will break for versions less than `v0.7.0`. A solution would be to have an if/else statement that checks the `vllm` version and falls back to the old logic. Personally I would just lower bound `vllm` since it's gonna be a pain to keep track of all the changes over time

Reply:
Would work for me